### PR TITLE
projects: Add missing fields to struct

### PR DESCRIPTION
**Pull request by radeksimko**
The original motivation was to add Project's `HtmlURL` field as we need it in https://github.com/terraform-providers/terraform-provider-github/pull/111
but I took some time to look at all missing fields in both Project & ProjectColumn, just for completeness.

The PR covers only one, small part of https://github.com/google/go-github/issues/884 - there is still more work to do before #884 can actually be closed - specifically around Project Collaborators and Team Projects.

### CODE REVIEW

File: github/projects.go [hunk 1]
```diff
@@ -74,6 +77,15 @@ type ProjectOptions struct {
 
 	// State of the project. Either "open" or "closed". (Optional.)
 	State string `json:"state,omitempty"`
```
Comment (gmlewis) [inline github/projects.go]: I believe all three of these optional parameters need to be pointers.
I think I made a mistake in the review of #746 and that we should have used `State *string` here, not `State string`. Can you please change that?

Also, please make the next two parameters pointers as well.

File: github/projects.go [hunk 2]
```diff
@@ -19,15 +19,18 @@ type ProjectsService service
...
-	OwnerURL  *string    `json:"owner_url,omitempty"`
-	Name      *string    `json:"name,omitempty"`
-	Body      *string    `json:"body,omitempty"`
-	Number    *int       `json:"number,omitempty"`
-	CreatedAt *Timestamp `json:"created_at,omitempty"`
-	UpdatedAt *Timestamp `json:"updated_at,omitempty"`
-	NodeID    *string    `json:"node_id,omitempty"`
+	ID         *int64     `json:"id,omitempty"`
+	URL        *string    `json:"url,omitempty"`
+	HtmlURL    *string    `json:"html_url,omitempty"`
```
Comment (gmlewis) [inline github/projects.go]: Let's be consistent with the rest of the naming in this repo and call this `HTMLURL` (for better or worse).

**Comment by radeksimko** [comment]
@gmlewis Thanks for the review. I believe I addressed all your comments. PTAL.

**Comment by radeksimko** [comment]
@gmlewis done, PTAL

**Pull request merged**